lib/views/player/components/bottom_action_component.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+
+class BottomActionComponent extends StatelessWidget {
+  const BottomActionComponent({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final _speedList = ['X1', 'X1.25', 'X1.5', 'X2', 'X0.6'];
+    var _currentSpeed = 'X1';
```

Comment:
Actually would be great if we could save the user's chosen speed and auto select that next time they load this screen?

Reply:
All these changes have been made in the player screen PR.